svc-api/handle/systems.go

```diff
@@ -131,6 +131,38 @@ func (sys *SystemRPCs) GetSystemResource(ctx iris.Context) {
 	ctx.Write(resp.Body)
 }
 
+// The method extract the session token,uuid and request url and creates the RPC request.
```

Comment:
no need to add a new  handler  `GetSystemResource` can be used please remove the newly added handler

Reply:
done